recipe/0489-multimedia-canvas/manifest.json

```diff
@@ -0,0 +1,57 @@
+{
+    "@context": "http://iiif.io/api/presentation/3/context.json",
+    "id": "{{ id.url }}",
+    "type": "Manifest",
+    "label": { "en": [ "Multimedia Canvas" ] },
+    "items": [
+      {
+        "id": "{{ id.path }}/canvas",
+        "type": "Canvas",
+        "height": 3024,
+        "width": 4032,
+        "duration": 120,
+        "items": [
+          {
+            "id": "{{ id.path }}/page/p1/1",
+            "type": "AnnotationPage",
+            "items": [
+              {
+                "id": "{{ id.path }}/annotation/p0001-image",
+                "type": "Annotation",
+                "motivation": "painting",
+                "body": {
+                  "id": "https://iiif.io/api/image/3.0/example/reference/918ecd18c2592080851777620de9bcb5-gottingen/full/max/0/default.jpg",
+                  "type": "Image",
+                  "format": "image/jpeg",
+                  "height": 3024,
+                  "width": 4032,
+                  "service": [
+                    {
+                    "id": "https://iiif.io/api/image/3.0/example/reference/918ecd18c2592080851777620de9bcb5-gottingen",
+                    "profile": "level1",
+                    "type": "ImageService3"
+                    }
+                  ]
+                },
+                "target": "{{ id.path }}/canvas"
+              },
+              {
+                "id": "{{ id.path }}/annotation/p0002-video q",
+                "type": "Annotation",
+                "motivation": "painting",
+                "body": {
+				  "id": "https://fixtures.iiif.io/video/indiana/donizetti-elixir/vae0637_accessH264_low_act_1.mp4",
+				  "type": "Video",
+				  "format": "video/mp4",
+				  "height": 360,
+				  "width": 480,
+				  "duration": 3971.24
```

Comment:
There is no selector on the resource, so because it's painted to: `t=27,57` I've assumed that the selector/cropping is `t=0,27`. Is that a correct assumption?

Reply:
Things are in-flight enough that it's a correct assumption given the current state of things, but it may become an incorrect one as I continue to edit. Right now, everything has been for placement only, to see whether I could get it syntactically valid and perhaps even working in a viewer. Now that you've shown me Theseus (thank you very much!) I can do something with the manifest and resources that might approach an authentic implementation of the use case. As part of that, I'll tighten up the syntax and make sure the property values are appropriate.